### PR TITLE
bug 1709910 ... but sometimes m5

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1709910

@wking, @jianlinliu, PTAL

I can't find an AMI for the other regions listed in https://github.com/openshift/installer/pull/1163/files#diff-bf7769ebd58e8a5c349b93522ca2ca02R7, so it looks like `eu-west-3` is the only region that this is true for in GA.

### CODE REVIEW

**Review by wking — APPROVED** [review]

File: modules/installation-creating-aws-control-plane.adoc [hunk 1]
```diff
@@ -176,6 +182,12 @@ and load balancing.
 . Copy the template from the *CloudFormation template for control plane machines*
 section of this topic and save it as a YAML file on your computer. This template
 describes the control plane machines that your cluster requires.
++
+[IMPORTANT]
+====
+If `m4` instance types are not available in your region, such as with
+`eu-west-3`, you must modify the CloudFormation template use `m5` types.
```
Comment (jianlinliu) [inline modules/installation-creating-aws-control-plane.adoc]: If want to use `m5` type, have to update MasterInstanceType.AllowedValues list together to include it in https://docs.openshift.com/container-platform/4.1/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-cloudformation-control-plane_installing-aws-user-infra

File: modules/installation-creating-aws-worker.adoc [hunk 2]
```diff
@@ -115,10 +115,22 @@ the CloudFormation template for the security group and roles.
...
+====
 
 . Copy the template from the *CloudFormation template for worker machines*
 section of this topic and save it as a YAML file on your computer. This template
 describes the networking objects and load balancers that your cluster requires.
++
+[IMPORTANT]
+====
+If `m4` instance types are not available in your region, such as with
+`eu-west-3`, you must modify the CloudFormation template use `m5` types.
```
Comment (jianlinliu) [inline modules/installation-creating-aws-worker.adoc]: If want to use `m5` type, have to update MasterInstanceType.AllowedValues list together to include it in https://docs.openshift.com/container-platform/4.1/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-cloudformation-control-plane_installing-aws-user-infra

**Comment by kalexand-rh** [comment]
@jianlinliu, will you please check again?

**Comment by kalexand-rh** [comment]
Jianlin approved the change in the source bug. Merging.

**Pull request merged**